VideoApi/VideoApi.Contract/Consts/HearingVenueNames.cs

```diff
@@ -0,0 +1,11 @@
+namespace VideoApi.Contract.Consts
+{
+    public static class HearingVenueNames
```

Comment:
Name it as ScottishHearingVenueNames ?

Reply:
i left this generic so later if we need to add more venue names const we can use this cs file

Reply:
That will work for now.  I'm thinking more for the future when there are a lot of venues and possibly venue properties too.